README.md

```diff
@@ -3,10 +3,11 @@
 Elixir LangChain enables Elixir applications to integrate AI services and self-hosted models into an application.
 
 Currently supported AI services:
+
 - OpenAI ChatGPT
 - OpenAI DALL-e 2 - image generation
 - Anthropic Claude
-- Google AI - https://generativelanguage.googleapis.com
+- Google AI - <https://generativelanguage.googleapis.com>
```

Comment:
Why the surrounding angle brackets?

Reply:
Ups - sorry, this is something that `mix format` has inserted, I didn't notice that. (I have a format-on-save setup in my editor). Should we remove this URL alltogether? The other providers don't have a URL, and btw this is not even a valid one (results in a 404).

Reply:
This commits fixes it: a97885899f32dc06773ef4cd386403e4cbe1d9af (but I'm not sure what's the interaction between the comments and the newly submitted commits)